tools/cldr-apps/src/main/webapp/about.jsp

```diff
@@ -117,5 +117,8 @@
 		<a href="./survey">Return to Survey Tool</a> |
 		<a href="./index.jsp">Return to CLDR Applications</a> |
 		<a target="_new" href="http://dev.icu-project.org/cgi-bin/cldrwiki.pl?SurveyToolHelp/About">Help on this page (in a new window)</a>
+
+		<!--  load bundle.js from webpack -->
+		<script src="./dist/bundle.js"></script>
 	</body>
```

Comment:
What about the modern "About" code in js/new/cldrAbout.js? Have this PR been tested with USE_DOJO_DEFAULT false?

Reply:
I have tested it with (and use it with) USE_DOJO_DEFAULT, this was just a place to cause the script to be loaded.  This jsp is still usable even with USE_DOJO_DEFAULT=false, all the better since this jsp will be going away.

Reply:
Shall we move js/new/* to js/src/*?

I had been thinking we'd move js/new/* to js/* as soon as we removed the dojo versions, but now we could move new to src, right? And see what happens with dist/bundle. Anyway can keep running directly from src if the bundle doesn't work yet

Reply:
> Shall we move js/new/* to js/src/*?

eventually, yes. I would probably do that component by component.  I was going to add Vue and such to the bundle in a followon PR

Reply:
Actually I didn't notice until now that this PR created a new js folder

* tools/cldr-apps/js

distinct from pre-existing

* tools/cldr-apps/src/main/webapp/js

So our current js files will migrate from

* tools/cldr-apps/src/main/webapp/js/new

to

* tools/cldr-apps/js/src

And the bundle will be created at

* tools/cldr-apps/src/main/webapp/dist

Will there then be no more tools/cldr-apps/src/main/webapp/js at all?

Reply:
> Will there then be no more tools/cldr-apps/src/main/webapp/js at all?
 
Eventually. But we need to make sure that components can actually function in that environment.